core/jvm/src/main/scala/scalaz/zio/RTS.scala

```diff
@@ -932,7 +884,7 @@ private object RTS {
                 }
             }
 
-            val finalizer = interruptStack
+            val finalizer = interruptStack(false)
```

Comment:
It looks like we're losing these finalizer failures. I'd put a big `FIXME` and we can solve it in a separate ticket. This one is big enough! 😆